app/tests/test_auth.py

```diff
@@ -0,0 +1,49 @@
+import json
+
+from django.test import TestCase
+from rest_framework.test import APIClient
+
+class UserTestCase(TestCase):
```

Comment:
We can also test with empty/ incomplete payload. (Ex: login with 'email' but provide not password attribute or vice versa).